### PR TITLE
doc: add minutes for 2017-12-05

### PR DESCRIPTION
**Pull request by gibfahn**
This is the first meeting to use the new auto-generated template, so we
should see what needs adding.

Fixes: https://github.com/nodejs/build/issues/1028

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2017-12-05.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+* Permission to start culling all pre-v4 infra and build logic [#926](https://github.com/nodejs/build/issues/926)
+* suggestion: investigate a commit-queue solution [#705](https://github.com/nodejs/build/issues/705)
+* [#1030] https://github.com/nodejs/build/issues/1030
+* Blue overview
+
+## Minutes
+
+### Provide read access to our ci.nodejs.org configs [#972](https://github.com/nodejs/build/issues/972)
+  * General agreement, just need to make sure no sensitive info will be exposed
+* Ubuntu non-LTS strategy [#967](https://github.com/nodejs/build/issues/967)
```
Comment (maclover7) [inline doc/meetings/build-wg-meeting-2017-12-05.md]: I think this and the following line need a `###` heading

File: doc/meetings/build-wg-meeting-2017-12-05.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  * CI Job tweaks
+  * core build scripts (Makefile, .gyp, and vcbuild)
+  * GCC upgrade investigation 
+* Rich Trott (@Trott)
+* Rod Vagg (@rvagg)
+* Gibson Fahnestock (@gibfahn)
+  * looked at OSX ansible stuff with George
+  * working on job to test cmake replacement for gyp
+
+
```
Comment (maclover7) [inline doc/meetings/build-wg-meeting-2017-12-05.md]: not a big deal, but should some of these extra lines be trimmed?

File: doc/meetings/build-wg-meeting-2017-12-05.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* Rich Trott (@Trott)
+* Rod Vagg (@rvagg)
+* Gibson Fahnestock (@gibfahn)
+  * looked at OSX ansible stuff with George
+  * working on job to test cmake replacement for gyp
+
+## Agenda
+
+* New OSX infra walkthrough [#1026](https://github.com/nodejs/build/issues/1026)
+* Provide read access to our ci.nodejs.org configs [#972](https://github.com/nodejs/build/issues/972)
```
Comment (mhdawson) [inline doc/meetings/build-wg-meeting-2017-12-05.md]: Need note that we deferred this to next time.

File: doc/meetings/build-wg-meeting-2017-12-05.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+* Ubuntu non-LTS strategy [#967](https://github.com/nodejs/build/issues/967)
+* Remove ARM64 Ubuntu 14.04 X-Gene servers from CI [#966](https://github.com/nodejs/build/issues/966)
+* Remove Fedora 22, 23, 24 [#962](https://github.com/nodejs/build/issues/962)
+* Remove Ubuntu 12.04 [#961](https://github.com/nodejs/build/issues/961)
+* Make it easier for people to join the Build WG [#941](https://github.com/nodejs/build/issues/941)
+* Permission to start culling all pre-v4 infra and build logic [#926](https://github.com/nodejs/build/issues/926)
+* suggestion: investigate a commit-queue solution [#705](https://github.com/nodejs/build/issues/705)
+* [#1030] https://github.com/nodejs/build/issues/1030
+* Blue overview
+
```
Comment (mhdawson) [inline doc/meetings/build-wg-meeting-2017-12-05.md]: add
  Jon gave us an overview of the Jenkins Blue UI and some work he's done on pipelines so far.

**Pull request merged**